javascript-challenges.test.js

```diff
@@ -0,0 +1,111 @@
+// ASSESSMENT 5: JavaScript Coding Practical Questions with Jest
+
+
+// Please read all questions thoroughly
+// Pseudo coding is REQUIRED
+// If you get stuck, please leave comments to help us understand your thought process
+
+// Use test driven development to complete the following questions
+// Add appropriate dependencies: $ yarn add jest
+
+// Reminder: The test will call your function
+// Run the file with the following command: $ yarn jest
+
+// --------------------1) Create a function that takes in a string and returns a string with a coded message. The coded message converts "a" to 4, "e" to 3, "i" to 1, and "o" to 0.
+
+// a) Create a test with expect statements using the variables provided.
+// b) Create the function that makes the test pass.
+
+const secretCodeWord1 = "Lackadaisical"
+// Expected output: "L4ck4d41s1c4l"
+const secretCodeWord2 = "Gobbledygook"
+// Expected output: "G0bbl3dyg00k"
+const secretCodeWord3 = "Eccentric"
+// Expected output: "3cc3ntr1c"
+
+describe("secretCodeWord1,2,3", () => {
+    it("takes in a string and returns the string with a coded message", () => {
+        expect(newCode(secretCodeWord1)).toEqual("L4ck4d41s1c4l")
+        expect(newCode(secretCodeWord2)).toEqual("G0bbl3dyg00k")
+        expect(newCode(secretCodeWord3)).toEqual("3cc3ntr1c")
+    })
+})
+
+
+const newCode = (input) => {
+    let newCode = ''
```

Comment:
Logic looks great.  Something to consider would be to make sure that you use unique names for your variables that do not match your function name.